spec/classes/minecraft_source_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'spec_helper'
+
+describe 'minecraft::source', :type => :class do
```

Comment:
I'm not 100% sure about this.  The `include ::minecraft` pre_condition will include `minecraft::source`.